Source/SwiftLintFramework/Models/Linter.swift

```diff
@@ -10,6 +10,14 @@ import Foundation
 import SwiftXPC
 import SourceKittenFramework
 
+private func correctableRules() -> [CorrectableRule] {
```

Comment:
Maybe instead of keeping a separate list we have to maintain we could `flatMap` enabled rules from the configuration `as? CorrectableRule` ?